sgkit_vcf/vcf_reader.py

```diff
@@ -48,7 +48,7 @@ def get_region_start(region: str) -> int:
 
 def vcf_to_zarr_sequential(
     input: PathType,
-    output: PathType,
+    output: Union[PathType, MutableMapping[str, bytes]],
```

Comment:
How did you land on `MutableMapping[str, bytes]` by the way?  I can't find anything to indicate that those were the only possible generic types.

Reply:
From https://zarr.readthedocs.io/en/stable/api/storage.html:

> Note that any object implementing the `MutableMapping` interface from the collections module in the Python standard library can be used as a Zarr array store, as long as it accepts string (str) keys and bytes values.